frontend/pages/project-developer/[id]/index.tsx

```diff
@@ -1,108 +1,126 @@
+  projectDeveloper,
+  enums,
+}) => {
+  const projectDeveloperTypeName = enums[EnumTypes.ProjectDeveloperType].find(
+    ({ id }) => id === projectDeveloper.project_developer_type
+  )?.name;
+
+  const funding = {
+    funded: projectDeveloper.projects.filter(({ received_funding }) => received_funding === true)
+      .length,
+    notFunded: projectDeveloper.projects.filter(
+      ({ looking_for_funding }) => looking_for_funding === true
+    ).length,
   };
 
-  const tagsGridRows: TagsGridRowType[] = [
+  const social = SOCIAL_DATA.map((item) => item.id)
+    .reduce((acc, social) => [...acc, { id: social, url: projectDeveloper[social] }], [])
+    .filter((social) => social.url);
+
+  const tagsRows: TagsGridRowType[] = [
     {
       title: 'Categories of interest',
       type: 'category',
-      tags: [
-        { id: 'tourism-and-recreation', name: 'Tourism & Recreation' },
-        { id: 'non-timber-forest-production', name: 'Non-timber forest production' },
-      ],
+      tags: enums[EnumTypes.Category].filter(({ id }) =>
+        projectDeveloper.categories?.includes(id as CategoryType)
+      ),
     },
     {
       title: 'Areas of work',
-      tags: ['Corazón Amazonía'],
+      tags: enums[EnumTypes.Mosaic].filter(({ id }) => projectDeveloper.mosaics?.includes(id)),
     },
     {
       title: 'Impact',
-      tags: ['Biodiversity', 'Social'],
+      tags: enums[EnumTypes.Impact].filter(({ id }) => projectDeveloper.impacts?.includes(id)),
     },
   ];
 
-  const projects = [...Array(6)].map((_, index) => {
-    return {
-      id: `project-${index}`,
-      name: `Circulo de Creaciones Cidaticas Circreadi ${index}`,
-      category: 'Tourism & recreation',
-      instrument: 'Grant',
-      amount: 25000,
-    };
-  });
+  const projects = projectDeveloper.projects.map((project) => ({
+    id: `project-${project.id}`,
+    slug: project.slug,
+    name: project.name,
+    category: enums[EnumTypes.Category].find(({ id }) => id === project.category)?.name,
+    instrument: enums[EnumTypes.InstrumentType]
+      .filter(({ id }) => projectDeveloper.projects[0].instrument_types?.includes(id))
+      .reduce((acc, instrument) => [...acc, instrument.name], [])
+      .join(', '),
+    amount: project.received_funding_amount_usd || 0,
+  }));
 
   return (
     <>
-      <Head title={`${aboutInfo.name} - ${aboutInfo.description}`} description={aboutInfo.text} />
+      <Head
+        title={`${projectDeveloper.name} - ${projectDeveloperTypeName}`}
+        description={projectDeveloper.about}
+      />
 
       <ProfileHeader
-        logo={aboutInfo.logo}
-        title={aboutInfo.name}
-        subtitle={aboutInfo.description}
-        text={aboutInfo.text}
-        website={aboutInfo.website}
-        social={aboutInfo.social}
-        contact={aboutInfo.contact}
-        numNotFunded={10}
-        numFunded={3}
+        logo={projectDeveloper.picture.medium}
```

Comment:
I'm wondering if [LET-344](https://vizzuality.atlassian.net/browse/LET-344) won't force you to change this path in the future.

Reply:
Is the logo loading for you? It always sends back a 404 to me with this error:
>"url" parameter is valid but upstream response is invalid

Reply:
> I'm wondering if [LET-344](https://vizzuality.atlassian.net/browse/LET-344) won't force you to change this path in the future.

If the backend changes the endpoint response yes, but I don't see why it would considering direct upload has been implemented already (in the backend). 

> Is the logo loading for you? It always sends back a 404 to me with this error:

It is not loading for me either. At the moment the image urls returned are 404s (possibly related to the direct upload?). That's what reminded me to implement the fallback in advance, so that backend changes related to this image don't stall the frontend in case something changes.